notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1404 @@
77rr22P+vPG2dRh5xFHjj9HV/WdI+1t+0ZIgqgTcZfnq3IqAIKAJpIpC137Q06571vJTAm/UW1PIrAoqAIqAIKAIlgIASeEugEbUKqx0BJfCu9ibQAlQQAjrgUEHAaraKgCKgCCgCikAeBJTAmwcgvVzlEIA4s/nmm1uyD5V/5JFH7HLXpQDE559/biAsQcCKMki7N910U9Ev2R1VB72WDgJZ89OUwGtM3bp1zWeffWY7AMTPLbfcskxngKzYsWNHe/6MM84wo0aNyknD/d27dzdTpkzJOc8BRMx77rnHdOrUybvGJAdRGz3ppJPs96Z3cdVO06ZNrZo3RFeUT3v37m1GjBhh/vvf/7rJ7PcTZQkie0Jg7dWrV6wJFQcccIAlbeZkHnEAofmSSy4xd9xxh/GrSUKsBKNhw4aZDTfc0OYyevRo07lzZ7t/zjnnmDvvvDMwdwiukG4xSNW1a9fOScfzrrzySgOx0W8QNl944QVLYOVZPDPKdt55Z0tMddMwMeOUU04xM2bMcE97+/Xr1zcPP/ywJV17J//Zcdv07rvvNqRt3769oU+5BvmasUrIwxCRTz31VEvWdtNAsibN7rvv7p6O3Hexe+aZZzzCq3vT8ccfb8aNG2dPXXXVVebqq692L9t9lywLaRZS9vDhw8ukc08MHTrU9O3b1ztVaP/wbgzZ4fnnn3++vUqfoG9gfO74/OazMWPGmJNPPjknWZK2loyS5JGPwBu3r0tZKmqrv2nJkFUCbzL89G5FQBFQBNJEIGu/aWnWPet5KYE36y2o5VcEFAFFQBFQBEoAASXwlkAjahVWOwIod1x++eVWFem4447LCZis9sJpARSBBAjogEMC8PRWRUARUAQUAUUgAQJK4E0Ant5akgigJAlxDbU+SEcDBgwIVbrLKgBvv/22JVOxZPhPP/1k1RMhEaFaiXpmy5Yts1o1LXfKCGTNT1MCr7HfW5D1MRRKUXT12+OPP27Jspw//fTTzf333+8l4TsQwu27775rz9EHIAHzXcE1jPEoIU5ynG+yuZD7UG1FKfXFF1/ktkA75phjzPjx43Ou/fzzz1YNXOrFxerVq9syobDqN9RuUfeNY7/88otViUUBN8r69etnBg8ebJM8+OCDFjcO+L2AjBxkEFY//PBDe4myQx4VY3KIS4KGHIzy+fLlyyWJVbFt2LChOfPMM3PayEvg7PCs999/3zsD+RoyNUrHUYayMorzEH1dc9t0v/3285Rh3TSyD3n42muvNaeddppVdZbz7pb2gqCKkm8c69Kli0csP+usswwkYtcgf9eoUcMjP9PH5syZ4yax+7SP3Nu/f39DX7rlllvKpHNPMIGlT58+9lR5+oebl38f9WbUlfn9xcCM/oRBbnf7iD0Z8I/P7wknnOBdSdrWZJQ0D/mMkxd9ftddd2XXWiF9Xe6pqK3+piVDVgm8yfDTuxUBRUARSBOBrP2mpVn3rOelBN6st6CWXxFQBBQBRUARKAEElMBbAo2oVVAEFAFFoIIQ0AGHCgJWs1UEFAFFQBFQBPIgoATePADpZUVAEVAEqjACWfPTlMCbnMD72GOPearjjRo1Mk8//bQlBUOYhHh7ww03mAYNGlhFWvlouGRPVL4fffRRuWS3LrlPLnTo0MGSRqtVq2YGDRpkJk+ebC/R5+bPn5+j1gqZEgVcbM899zQQGCGNQq4877zzPAIkky4gCzIhYZtttrHp8/279NJLzfXXX2+TobY7cOBAc9hhhxnIxi+//LIl7UI87dq1q1UeJmEaBN7ddtvNCGmYulGPNddc05KuUTiGVPvKK6+Ybbfd1ixZssSSeT/++GOrQkwZUL699dZb2bUG8VMUblGMBR/uw8hjyJAhlpj9559/WqVc6i1quqjOIxjgkmvdNrWZrPqHyi7lRB0YDIKUhyGM9+zZ07Rt29ZMnTrVKgyLqjHEWVSU49iECRPM0UcfbZPWqlXLLF68OOc2yLr77LOPd45+A2HdrzgNLhBjsbfeestiwRZsIAljNWvWzCFIQ1jeeOON7bXy9A97Y8A/FIovuOACTwUfwjYTamgrsZkzZ9oJRJCN33jjDXsacjSfIWyNNdYwKEyvu+669jiNtk4jD/cz7ifwFtLXbaUq8J/+piUDVwm8yfDTuxUBRUARSBOBrP2mpVn3rOelBN6st6CWXxFQBBQBRUARKAEElMBbAo2oVVAEFAFFoIIQ0AGHCgJWs1UEFAFFQBFQBPIgoATePADpZUVAEVAEqjACWfPTlMCbnMCL0iwkXWzUqFHmjDPOyPsJcMmecQi8rsIpmUMqrV+/voGgikGoveSSS+w+/5o0aWLJl/RHP7kXxVrIuuSBffnll5akaQ/y/IPYCRH2P//5j005duxYc/zxx+fchRo75YEsesQRR9hrSQm8ECY32mgjA7EVsvEXX3yR88ywg3/9619WJZ3rLVq0MJA9gwyl4Msuu8xeqrNKWR0isJ/QvGDBAgNRddmyZTYdq31dd911XnZum3IS0jaqxqjeirVr184jXnNu6623NtOnT7eYSprevXt7ROODDz7YTJs2TS5FbsGIZ/3+++82HUTXxo0be/dA+qbMrtEuKNqK0VcoN+bHGczBBoNg+sEHH9h99195+4fkcdRRR5mlS5fa/gVBmr4kBmkYcrxLQpZrbI899lh7nX0wA7sgS6Ot08gjjMBb3r4eVNc0zulvWjIUlcCbDD+9WxFQBBSBNBHI2m9amnXPel5K4M16C2r5FQFFQBFQBBSBEkBACbwl0IhaBUVAEVAEKggBHXCoIGA1W0VAEVAEFAFFIA8CSuDNA5BeVgQUAUWgCiOQNT9NCbzJCbyQOK+88krb6yHjomgb1A/cj4VL9sxH4IUcfPHFF7u3232XkHn++ed7pE9Iruuvv74lQaKuKqqxbgaQWSGpYjNmzDAtW7Z0L4fujx492nTu3Nleh0D83nvvhaZ1LyQl8EI2RuF1xYoVVnUXYi7Pz2dxCbwQU4U
```

Comment:
I've updated the text to highlight that it's the left fork on the right branch, and that the function is within wfc3tools:

Note that the left fork under the drizzle level branch is an advanced workflow that involves drizzling the empirical models provided by the WFC3 team into FLC files. This requires installing a custom version of [wfc3tools](https://github.com/Vb2341/wfc3tools), running the make_model_star_image() function within wfc3tools, and then stacking the drizzled models. This functionality is under development and not yet officially supported.

Reply:
Thanks. Could you directly link to the function or be more explicit as to where to find it? I see you linked Varun's fork, but is that function in the master branch or another? What module does it exist in (e.g. `utils.py`)? I'm having a hard time getting there myself, but maybe I'm missing something in plain sight 🤦🏾‍♂️

Reply:
The reason you could not find it is because somehow I linked wfc3tools instead of wfc3_photometry 🙃 

I have updated the link to: [wfc3_photometry](https://github.com/spacetelescope/wfc3_photometry).

The text now reads:

"This requires installing the [wfc3_photometry](https://github.com/spacetelescope/wfc3_photometry) package, running the make_model_star_image() function within PSFUtils.py, and then stacking the drizzled models. This functionality is under development and not yet officially supported."